### DISCUSSION TITLE
Notification on Issues

### DISCUSSION THREAD

**Discussion by maniacx** [community/community#67911] [Mobile] [answered]
If an issues is raised or a commented, on the project I am maintaining,send a notification and give us an option to enable/disable in settings on GitHub Mobile app android

**Comment by stevepopovich** [comment, marked as answer]
Hi @maniacx you should be able to get push notifications and GitHub notifications for issue activity. You can check your settings: 
On Mobile: 
Profile -> Settings -> Configure Notifications

You can also make sure you are watching activity on a repository: 
Go to any repository -> Bell icon

If this doesn't help, please let me know!

**Reply by Viralface** [reply]
@maniacx

**Comment by maniacx** [comment]
Thank you @stevepopovich.
I found the option in the Bell icon was set to wrong one. (I may have done it months ago and didnt realise it)
Apologise for wasting your time, but I was lost and blind to see the bell icon present next to my repository in the mobile app.
I have tested it but i think it should work now.
I will close this issue.
Thank you again.

**Reply by stevepopovich** [reply]
Thanks for getting back and I am glad the issue is resolved!
